documentation/src/main/asciidoc/topics/proc_using_kyro_marshaller.adoc

```diff
@@ -0,0 +1,27 @@
+[id='kryo_marshalling']
+= Using the Kryo Marshaller
+{brandname} provides a marshalling implementation that uses Kryo libraries.
+
+.Procedure
+
+. Implement a service provider for the `SerializerRegistryService.java` interface.
+. Place all serializer registrations in the `register(Kryo)` method; where serializers are registered with the supplied `Kryo` object using the Kryo API, for example:
++
+----
+kryo.register(ExampleObject.class, new ExampleObjectSerializer())
+----
++
+. Specify the full path of implementing classes in your deployment JAR file within:
++
+----
+META-INF/services/org/infinispan/marshaller/kryo/SerializerRegistryService
+----
+
+//Community only
+ifndef::productized[]
+{brandname} provides a link:http://central.maven.org/maven2/org/infinispan/infinispan-marshaller-kryo-bundle/{infinispanversion}/infinispan-marshaller-kryo-bundle-{infinispanversion}.jar[Kryo Bundle] that includes all runtime class files for the Kryo marshalling implementation. Download the JAR and add it to the `server/lib` directory of your {brandname} servers.
```

Comment:
This only applies to server mode. It's possible to utilise the Kryo marshaller in Embedded mode by just adding the `infinispan-marshaller-kryo` jar as a dependency and then specifying the `org.infinispan.marshaller.kryo.KryoMarshaller` class as the marshaller.